code/modules/power/lighting/lights.dm

```diff
@@ -111,6 +111,20 @@
 	nightshift_range = 4
 	nightshift_power = 0.4
 
+/obj/item/light/bulb/strong
```

Comment:
is this supposed to be unobtainable? if no, please add them to the lathe (https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/5283 for what to do)

Reply:
Yes, these are only used in one type of street lamp that has a small bulb sprited, but needs the 8 tile radius of a strong bulb/light tube